Tools/CISupport/build-webkit-org/steps.py

```diff
@@ -56,6 +56,10 @@
 THRESHOLD_FOR_EXCESSIVE_LOGS = 1000000
 MSG_FOR_EXCESSIVE_LOGS = f'Stopped due to excessive logging, limit: {THRESHOLD_FOR_EXCESSIVE_LOGS}'
 
+DNS_NAME = CURRENT_HOSTNAME
```

Comment:
should we just change CURRENT_HOSTNAME variable instead of creating another variable (DNS_NAME)

Reply:
Possibly? I wanted to create minimal disruption, so I'm happy with that change in the future, just trying to unbreak things at the moment